### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: i18n/fr-CH/fr_CH.po [hunk 1]
```diff
@@ -3207,7 +3207,7 @@ msgstr "Gestion des modèles de rapports"
 
 #: app/scripts/reports/detail/reportsdetail.html:17
 msgid "Verändern"
-msgstr ""
+msgstr "Pauser"
```
Comment (AlwinEgger) [inline i18n/fr-CH/fr_CH.po:3210]: This is wrong, no?

**Review by AlwinEgger — CHANGES_REQUESTED** [review]
I've changed it on crowdin.

File: i18n/fr-CH/fr_CH.po (hunk 1 above)
Comment (mcmpp) [reply i18n/fr-CH/fr_CH.po:3210]: thanks, got inspired by the other languages. So, I assume I will need to change the others too...

Comment (AlwinEgger) [reply i18n/fr-CH/fr_CH.po:3210]: I guess.

**Pull request closed without merging**